CHANGES.rst

```diff
@@ -19,6 +19,8 @@ Other
 1.4.4 (2023-09-15)
 ==================
 
+- Added ``alignment`` sub-package. [#179]
```

Comment:
This needs to be moved to the unreleased section.

Reply:
Now it's done.